common/tasks/rosa-hcp/README.md

```diff
@@ -0,0 +1,22 @@
+# ROSA HCP Cluster
+
+## ROSA with HCP Prerequisites
+https://docs.openshift.com/rosa/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-hcp-prereqs
+
+
+## How to use the tasks
+[Here](hcp-cluster-test-pipeline.yaml) is an example about how to use tasks to provision/deprovision ROSA HCP cluster
+
+### Secrets
+You need to create a secret named `hcp-cluster-secre` in Konflux, which includes the following data.
+
+* AWS_ACCOUNT_ID
+* AWS_OIDC_CONFIG_ID
+* INSTALL_ROLE_ARN
+* OPERATOR_ROLES_PREFIX
+* SUBNET_IDS
+* SUPPORT_ROLE_ARN
+* WORKER_ROLE_ARN
+* aws_access_key_id
```

Comment:
can we use uppercase for all environments?

Reply:
fixed